config/production_config.json

```diff
@@ -32,10 +33,16 @@
 	"MAIL_DB_NAME": "mail",
 	"EVENT_DB_NAME": "event",
 	"STAT_DB_NAME": "stat",
+	"NOTIFICATIONS_DB_NAME": "notifications",
 
 	"S3_REGION": "us-east-1",
 	"S3_BUCKET": "hackillinois-upload-2019",
 
+	"SNS_REGION": "us-east-1",
+
+	"ANDROID_PLATFORM_ARN": "arn:aws:sns:us-east-1:256758753660:app/GCM/android",
```

Comment:
Leave the arns out of the prod config. We'll inject the productions ones using env vars.